portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,11 +44,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Get response from the form
+    //Get response from the form
     String text = request.getParameter("text-input");
     if (text != null && !text.isEmpty()) {
       myComments.add(text);
     }
+    Entity comment = new Entity("Comment");
+    comment.setProperty("text", text);
+    DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
nit - consider moving this to be an member variable of the class. It won't matter much for this instance, but in the future when you'll be reading an writing from the datastore it will be nice to have a single reference to it instead of having to get one every time.

Reply:
ah thanks for the note! will update!